Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m

```diff
@@ -52,21 +55,27 @@ - (void)setupAccessoryViews
 {
     //Attach given views, or generate default views.
     if (!self.headerView) {
-        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:(CGRect){0, -30, self.tableView.frame.size.width, 30}];
+        CGRect headerFrame = (CGRect){0, -DefaultAccessoryHeight, self.tableView.frame.size.width, DefaultAccessoryHeight};
```

Comment:
I'd strongly prefer either `CGRectMake()` or `(CGRect){ .origin = …, .size = … }` (using the labels on this kind of literal/initializer), here and wherever else this construct is used.